Filter bad stop times

weatherpattern opened the pull request:
**Summary:**
RTDII-194 https://issuetracker.camsys.com/browse/RTDII-194
 
`GtfsReader` throws an exception if a `stop_time` is missing `stop_id` and `start_service_area_id` and `end_service_area_id`

**Expected behavior:** 

If `stop_times.txt` has a row that is missing`stop_id` and `start_service_area_id` and `end_service_area_id` throw a `NoSuchElementException` exception

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ X] Run the unit tests with `mvn test` to make sure you didn't break anything
- [X] Format the title like "Fix #<issue_number> - short description of fix and changes"
- [x ] Linked all relevant issues

Commit:
RTDII-194 throw exception for bad stoptime

Commit:
ignor iml files

Commit:
remove .iml from branch

mmaranda-cs reviewed (commented):
Are there tests written for this change to GtfsReader.java?

Commit:
fix PR#6 and PR#7 comments

weatherpattern commented:
Added a new test and data test to check if the expected `NoSuchElementException` is thrown.
Added commented to explain why `stop_id` in `stoptimes.txt` is now option and what is the new `stop_id` requirement for gtfs-flex.
